movie-browser/src/helpers/movieData.js

```diff
@@ -0,0 +1,21 @@
+// helper that filters movies data and returns an object
```

Comment:
although i like this method what i meant was a completely different thing. It was more like this:

```
ON API.JS:

export const fetchData = apiURL => {
  return axios
    .get(apiURL)
    .then(res =>{ movies: res, featuredMovies: res.filter(movie => movie.rt_score > 94) })
    .catch(err => console.log(err));
};

your fetchMovies can return everything at once in an object.
```

Reply:
But this is not a blocker. It was just to show u what i meant. You can merge if you want. I will approve.

Reply:
I better understand what you mean now. Doing it in fetchData is more direct and involves less code.